index.js

```diff
@@ -2,11 +2,11 @@
 
 var assert = require('assert');
 
-var WM = require('es6-weak-map');
+var WeakMap = require('es6-weak-map');
```

Comment:
Typo? Guessing you meant to suggest keeping it named `WM` s before. I don't see how naming it `WeakMap` would clobber the global, since it is local var. But it did shadow that global, where available. Anyway, I amended the PR.